users/user-guide-query/multi-stage-query/operator-types/aggregate.md

```diff
@@ -0,0 +1,194 @@
+GROUP BY a.col1
+```
+
+## Stats
+### executionTimeMs
+Type: Long
+
+The summation of time spent by all threads executing the operator.
+This means that the wall time spent in the operation may be smaller that this value if the parallelism is larger than 1.
+Remember that this value is affected by the number of received rows and the complexity of the aggregation function.
+
+### emittedRows
+Type: Long
+
+The number of groups emitted by the operator.
+A large number of emitted rows can indicate that the query is not well optimized.
+
+Remember that the number of groups is limited by the `num_groups_limit` hint and a large number of groups can lead to
+high memory usage and slow queries.
+
+### numGroupsLimitReached
+Type: Boolean
+
+This stat is set to true when the number of groups exceeds the limit defined by the `num_groups_limit` hint.
+In that case, the query will not fail but will return partial results, which will be indicated by the global 
+`partialResponse` stat.
+
+## Explain attributes
+The aggregate operator is represented in the explain plan as a `LogicalAggregate` explain node.
+
+Remember that these nodes appear in pairs: First in one stage where the aggregation is done in parallel and then in the
+upstream stage where the partial results are merged.
+
+### group
+Type: List of Integer
+
+The list of columns used in the `group by` clause.
+These numbers are 0-based column indexes on the virtual row projected by the upstream.
+
+For example the explain plan:
+```
+LogicalAggregate(group=[{6}], agg#0=[COUNT()], agg#1=[MAX($5)])
+  LogicalTableScan(table=[[default, userAttributes]])
+```
+
+Is saying that the `group by` clause is using the column with index 6 in the virtual row projected by the upstream.
+Given in this case that row is generated by a table scan, the column with index 6 is the 7th column in the table
+as defined in its schema.
+
+### agg#N
+Type: Expression
+
+The aggregation functions applied to the columns.
+There may be multiple `agg#N` attributes, each one representing a different aggregation function.
+
+For example the explain plan:
+```
+LogicalAggregate(group=[{6}], agg#0=[COUNT()], agg#1=[MAX($5)])
+  LogicalTableScan(table=[[default, userAttributes]])
+```
+
+Has two aggregation functions: `COUNT()` and `MAX()`.
+The second is applied to the column with index 5 in the virtual row projected by the upstream.
+Given in this case that row is generated by a table scan, the column with index 5 is the 6th column in the table
+as defined in its schema.
+
+## Tips and tricks
+
+### Try to not use aggregation functions that cannot be parallelized
+For example, it is recommended to use one of the different hyperloglog flavor instead of `count(distinct)` when
+the cardinality of the data or their size.
+
+For example, it is cheaper to execute `count(distinct)` on an int column with 1000 distinct values than on a column that
+stores very long strings, even if the number of distinct values is the same.
```

Comment:
this example also is kinda confusing in the context of "try not to use aggregation functions that cannot be parallelized"

Reply:
Maybe we should rephrase it as _be aware of the amount of bytes you are shuffling or sending between stages_.